worktree/setup.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Include utils
+source ../utils/utils.sh
+
+kata="worktree"
```

Shellcheck (0.11.0)

[warning] kata appears unused. Verify use (or export if used externally).

(SC2034)